language-design.md

```diff
@@ -74,6 +123,23 @@ the commenters?
 
 **Response**
 
+The comments on the Pavlus article generally seem to suggest that the article does not take many factors into consideration when claiming that programming languages should be more semantically intuitive. Pavlus insists that existing languages are very difficult to understand and have sacrificed usability for succinctness. He cites Perl for its confusing syntax and uses it as an example of how inaccessible programming languages are to the average person. 
+
+In order to address the conflict in the comments, we must look at the distinction between professional programmers and everyone else. Pavlus’ article says,
+
+> Unless you’ve had specialized training, looking at lines of code is like reading hieroglyphs, only less intuitive. [Pavlus, 2012]
+
+This defines professional programmers as those who have had specialized training and are thus able to understand code. Keeping this definition in mind, the majority of comments seemed to sympathize with professional programmers. While Pavlus mentions the need for a more intuitive programming language, the commenters cite the convenience of efficient syntax in writing professional programs. One commenter, Mjaksen, writes the following:
+
+> True but irrelevant. C/C++ *IS* a terrible language to express oneself in, but it's pretty good at 
+> organizing large software projects and interfacing with all the link libraries and shared libraries 
+> that it needs to get the job done… I have *never* seen a language that is concise, contextually 
+> clear, and easy to read, that can also organize code/objects/threads/errors and place nice with 
+> the os/shared libs. [Pavlus, 2012]
+
+The point made here is that it is difficult to have the best of both worlds. Programming languages are designed with efficiency in mind and make sacrifices in terms of readability.
+
+We agreed with the majority of the commenters in the article that there is a reason why current programming languages are not easy to read. Professional programmers are trained to work in these languages to maximise their output and the effectiveness of their programs. It is also important to consider the needs of those who are not professional programmers. They may have no formal training in programming languages. Pavlus advocates for this group of people but ignores the fact that computer science is far more than syntax knowledge. Computer science involves knowing how to design programs intelligently, regardless of specific language inconsistencies. Even if an efficient and semantically accessible programming language were to exist, it would still require specialized training to be able to write complex programs. 
 
```

Comment:
Computer science is much more than syntax knowledge--agreed. However, if you have strong logical reasoning but cannot figure out how to express yourself, it does not matter how smart or accomplished of a programmer you are.